### PR TITLE
fix: kubernetes deploy example compatiable with distroless image

### PR DESCRIPTION
**Pull request by everpcpc**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Make kubernetes deploy example compatiable with distroless image

## Changelog

- Documentation

## Related Issues

Fixes #issue

### CODE REVIEW

**Comment by BohuTANG** [comment]
@mergify update

File: scripts/kubernetes/meta-standalone.yaml [hunk 1]
```diff
@@ -21,20 +21,11 @@ spec:
           - "meta-service.databend-system.svc.cluster.local"
       containers:
         - name: databend-meta
-          image: datafuselabs/databend-meta:v0.6.97-nightly
-          command: ["/bin/bash", "-c"]
+          image: datafuselabs/databend-meta:v0.6.100-nightly
```
Comment (ZhiHanZ) [inline scripts/kubernetes/meta-standalone.yaml:24]: I think distroless image has different name? it is normal image

Comment (everpcpc) [reply scripts/kubernetes/meta-standalone.yaml:24]: we could wait until tomorrow for the new nightly image

Comment (ZhiHanZ) [reply scripts/kubernetes/meta-standalone.yaml:24]: LGTM

**Review by ZhiHanZ — APPROVED** [review]

**Pull request merged**